relay/server.go

```diff
@@ -136,14 +158,58 @@ func (s *Server) GetChunks(ctx context.Context, request *pb.GetChunksRequest) (*
 
 	// Future work:
 	//  - authentication
-	//  - global throttle
-	//  - per-connection throttle
 	//  - timeouts
 
 	if len(request.ChunkRequests) <= 0 {
 		return nil, fmt.Errorf("no chunk requests provided")
 	}
+	if len(request.ChunkRequests) > s.config.MaxKeysPerGetChunksRequest {
+		return nil, fmt.Errorf(
+			"too many chunk requests provided, max is %d", s.config.MaxKeysPerGetChunksRequest)
+	}
+
+	// Future work: client IDs will be fixed when authentication is implemented
+	clientID := fmt.Sprintf("%d", request.RequesterId)
```

Comment:
do we allow unauthenticated traffic or all requests need to be signed (for MVP)?

Reply:
My plan was to only permit authenticated traffic for `GetChunks()`. Is there any use case for allowing unauthenticated traffic? I can add it if you think it would be helpful.